lib/api/webSockets.js

```diff
@@ -54,6 +57,15 @@ function connect(connectionOps = {}) {
 			);
 		}
 
+		if (content.type === 'console' && content.data) {
+			const method = {
+				info: 'info',
+				log: 'log',
+			}[content.subtype] || 'debug';
+
+			return logger[method](`|L| ${method}:`, content.data.join('\n'));
```

Comment:
now that we introduce more type markup, I think it should have been placed in constants somewhere, before we make too much mess. Plus, please see the feedback in JIRA.